.github/workflows/build-docker-image.yml

```diff
@@ -67,12 +74,13 @@ jobs:
           # generate Docker tags based on the following events/attributes
           tags: |
             type=schedule
+            type=sha
             type=ref,event=branch
             type=ref,event=pr
             type=semver,pattern={{version}}
             type=semver,pattern={{major}}.{{minor}}
-            type=semver,pattern={{major}}
-            type=sha
+            # disabled if major zero
+            type=semver,pattern={{major}},enable=${{ !startsWith(github.ref, 'refs/tags/v0.') }}
```

Comment:
I'm confused about skipping major version zero.

Zebra is currently at major version 1, so this condition will never apply?
Also this isn't listed in the PR description.

Reply:
This is practically just skipping major versions if the tags are, for example `v0.1.0` which we might use for testing or other purposes. It's basically skipping every tag starting with `v0.`

Reply:
This doesn't match Rust conventions, it's common for Rust crates to stay at `v0.*` for years.

I think it would be very surprising to have this skipped, particularly if it is undocumented.

Reply:
Fixed by [ci(build): allow v0 as a major version tag](https://github.com/ZcashFoundation/zebra/pull/4939/commits/6c3e27877f5754fdcb8096126bdb2e3e6fd833e8)